src/univalue/univalue.cpp

```diff
@@ -175,21 +182,21 @@ bool UniValue::checkObject(const std::map<std::string,UniValue::VType>& t)
 const UniValue& UniValue::operator[](const std::string& key) const
 {
     if (typ != VOBJ)
-        return nullValue;
+        return NullUniValue;
 
     int index = findKey(key);
     if (index < 0)
-        return nullValue;
+        return NullUniValue;
 
     return values[index];
 }
 
 const UniValue& UniValue::operator[](unsigned int index) const
```

Comment:
This is still present in [latest upstream](https://github.com/jgarzik/univalue/blob/master/lib/univalue.cpp#L252), so should be turned into a PR there.